Smooch.postman_collection.json

```diff
@@ -17146,6 +17146,119 @@
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"{{apiVersion}}",
+										"apps",
+										"{{appId}}",
+										"integrations",
+										"{{integrationId}}",
+										"messageTemplates"
+									],
+									"query": [
+										{
+											"key": "name",
+											"value": null,
+											"description": "When specified, returns all the translations for a single message template.",
+											"disabled": true
+										},
+										{
+											"key": "limit",
+											"value": null,
+											"description": "When specified, limits the number of message templates to return per page.",
+											"disabled": true
+										},
+										{
+											"key": "before",
+											"value": null,
+											"description": "When specified, returns the message templates on the previous page. (eg: MjUZD)",
+											"disabled": true
+										},
+										{
+											"key": "after",
+											"value": null,
+											"description": "When specified, returns the message templates on the next page. (eg: NDkZD)",
+											"disabled": true
+										}
+									]
+								},
+								"description": "Lists all WhatsApp message templates in the account"
+							},
+							"response": []
+						},
+						{
+							"name": "Create Message Template",
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"category\": \"ALERT_UPDATE\",\n    \"name\": \"shipping_notification_5000\",\n    \"language\": \"en_US\",\n    \"components\": [{ \"type\": \"BODY\", \"text\": \"Powerman has shipped your order!\" }]\n}\n",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{url}}/{{apiVersion}}/apps/{{appId}}/integrations/{{integrationId}}/messageTemplates",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"{{apiVersion}}",
+										"apps",
+										"{{appId}}",
+										"integrations",
+										"{{integrationId}}",
+										"messageTemplates"
+									]
+								},
+								"description": "Creates a new WhatsApp message template."
+							},
+							"response": []
+						},
+						{
+							"name": "Delete Message Template",
+							"request": {
+								"method": "DELETE",
+								"header": [],
+								"url": {
+									"raw": "{{url}}/{{apiVersion}}/apps/{{appId}}/integrations/{{integrationId}}/messageTemplates/{{name}}",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"{{apiVersion}}",
+										"apps",
+										"{{appId}}",
+										"integrations",
+										"{{integrationId}}",
+										"messageTemplates",
+										"{{name}}"
+									]
+								},
+								"description": "Deletes a message template."
+							},
+							"response": []
```

Comment:
Should I put a sample response?

Reply:
I'd just stay consistent with the rest of the collection.